Update Wording in User Add doc (#2168)
astronomer/docs: astro/add-user.md

    ```sh
    sh path/to/add-users.sh path/to/users.txt
    ```

> 6. Optional. To use this script as part of a CI/CD pipeline, create a Workspace API token and specify the environment variable `ASTRO_API_TOKEN=<your-token>` in your CI/CD environment. See [Manage Workspace API tokens](workspace-api-tokens.md). Note that you can use Workspace API tokens to manage users at the Workspace level but not at the Organization level.

Reviewer: ```suggestion
6. (Optional) To use this script as part of a CI/CD pipeline, create a Workspace API token and specify the environment variable `ASTRO_API_TOKEN=<your-token>` in your CI/CD environment. See [Manage Workspace API tokens](workspace-api-tokens.md). Note that you can use Workspace API tokens to manage users only at the Workspace level.
```
